runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/process.json

```diff
@@ -421,6 +421,9 @@
     "Process.Details.Modeler.Actions.Desktop.TakeScreenshot.ImagePath": "Ścieżka do obrazu",
     "Process.Details.Modeler.Actions.Desktop.ReadTextFromImage.Label": "Odczytaj tekst z obrazu",
     "Process.Details.Modeler.Actions.Desktop.ReadTextFromImage.Language": "Język tekstu",
+    "Process.Details.Modeler.Actions.VBasic.RunMacro.Label": "Uruchom skrypt",
+    "Process.Details.Modeler.Actions.VBasic.RunMacro.Input.ScriptPath": "Ścieżka do skryptu",
+    "Process.Details.Modeler.Actions.VBasic.RunMacro.Input.ScriptArguments": "Argumenty skryptu Na przykład: arg1,arg2",
```

Comment:
form of provided arguments should be described in the info icon next to the field. In labels keep just the name of the field